invitations/service_test.go

```diff
@@ -68,7 +85,8 @@ func TestSendInvitation(t *testing.T) {
 			req:         validInvitation,
 			err:         svcerr.ErrAuthentication,
 			authNErr:    svcerr.ErrAuthentication,
-			domainErr:   nil,
+			domainErr:   svcerr.ErrAuthorization,
+			domainErr1:  nil,
```

Comment:
Make this error and the corresponding request more descriptive. e.g. `domainMembershipReq`.